colabs/pyg/Graph_Classification_with_PyG_and_W&B.ipynb

```diff
@@ -1,12 +1,13 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.        wandb.watch(model)
</pre><p>lets remove watch, I don't think its worth the hassle and slowness it can cause</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wandb/examples/pull/366/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>